### PR TITLE
finish out libnetcdf

### PR DESCRIPTION
**Pull request by ocefpaf**
This is safe to be merged b/c the remaining packages on the migration are probably too broken to build.

### CODE REVIEW

**Comment by ocefpaf** [comment]
Thanks!

**Comment by CJ-Wright** [comment]
Sure, although we should note those packages which we consider "too broken to build" somehow

**Comment by ocefpaf** [comment]
> Sure, although we should note those packages which we consider "too broken to build" somehow

On it in their feedstock. We'll probably archive them but it is pending the team's decision.

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -224,8 +224,6 @@ pin_run_as_build:
     max_pin: x.x
   libmatio:
     max_pin: x.x
-  libnetcdf:
-    max_pin: x.x.x
```
Comment (xylar) [inline recipe/conda_build_config.yaml]: Was there a discussion that it's no longer necessary to pin libnetcdf build as run?  I would have thought this needed to be at least `x.x`.  That was my understanding of this comment: https://github.com/conda-forge/libnetcdf-feedstock/pull/90#issuecomment-558239517

Comment (xylar) [reply recipe/conda_build_config.yaml]: Maybe it's fine.  https://abi-laboratory.pro/index.php?view=timeline&l=netcdf shows 100% backwards compatibility for all the releases we're likely to have on conda-forge.

Comment (ocefpaf) [reply recipe/conda_build_config.yaml]: @xylar with `run_exports` on the feedstock we don't need this here but the same pin still applies.

**Pull request merged**